pkg/helpers/helper.go

```diff
@@ -7,41 +7,77 @@ import (
 	"regexp"
 	"strings"
 
+	"github.com/fatih/color"
 	"github.com/hashicorp/go-version"
 )
 
+// ColoredVersion returns a colored version string
+func ColoredVersion(version string) string {
+	if IsPreReleaseVersion(version) {
+		return color.YellowString(version)
+	}
+	return color.CyanString(version)
+}
+
+// ColoredInstallHelper returns a colored install helper string
+func ColoredInstallHelper(version string) string {
+	return color.CyanString(fmt.Sprintf("`tfversion install %s`", version))
+}
+
+// ExitWithError prints an error message and exits with status code 1
+func ExitWithError(message string, err error) {
+	fmt.Printf("%s %s: %s\n", color.HiRedString("error:"), message, err)
+	os.Exit(1)
+}
+
+// ErorWithHelp returns an error with a help message
+func ErorWithHelp(help string) error {
```

Comment:
Small typo: Error > Eror

Reply:
Fixed in: [c7cc9b2](https://github.com/tfversion/tfversion/pull/15/commits/c7cc9b26c65f80649b04487f99108b1bacf6861a)

Reply:
lol derp typo